common/scala/src/main/scala/whisk/common/TransactionId.scala

```diff
@@ -53,11 +53,11 @@ case class TransactionId private (meta: TransactionMetadata) extends AnyVal {
    * @param message An additional message that is written into the log, together with the other information.
    * @param logLevel The Loglevel, the message should have. Default is <code>InfoLevel</code>.
    */
-  def mark(from: AnyRef, marker: LogMarkerToken, message: String = "", logLevel: LogLevel = InfoLevel)(
+  def mark(from: AnyRef, marker: LogMarkerToken, message: String = "", logLevel: LogLevel = DebugLevel)(
     implicit logging: Logging) = {
 
     if (TransactionId.metricsLog) {
-      logging.emit(logLevel, this, from, createMessageWithMarker(message, LogMarker(marker, deltaToStart)))
+      logging.emit(InfoLevel, this, from, createMessageWithMarker(message, LogMarker(marker, deltaToStart)))
```

Comment:
a comment here is advisable: even though the mark log message is potentially logged at debug level, the time stamp is emitted at info level.